src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -1635,11 +1647,20 @@ namespace RoomAir {
                     roomAFNNodeHVAC.SupplyFraction = ipsc->rNumericArgs(iEquipArg);
                     roomAFNNodeHVAC.ReturnFraction = ipsc->rNumericArgs(iEquipArg);
 
+                    // get equipment index
+                    int EquipIndex = 0;
+                    for (int thisZoneEquipNum = 1; thisZoneEquipNum <= state.dataZoneEquip->ZoneEquipList(iZone).NumOfEquipTypes;
+                         ++thisZoneEquipNum) {
+                        if (Util::SameString(state.dataZoneEquip->ZoneEquipList(iZone).EquipName(thisZoneEquipNum), roomAFNNodeHVAC.Name)) {
+                            EquipIndex = state.dataZoneEquip->ZoneEquipList(iZone).EquipIndex(thisZoneEquipNum);
+                            break;
+                        }
+                    }
                     IntEquipError = CheckEquipName(state,
```

Comment:
Minor, but the name `IntEquipError` is misleading, better would be `IntEquipFound`?

Reply:
Done!